### ISSUE TITLE
UnsupportedDocumentTypeException for "application/PDF"

### ISSUE THREAD

**Issue by klr8** [e-Contract/dssp#8]
When uploading data with MIME type "application/PDF" to https://doccle.e-contract.be/dss-ws we get the following exception:
```
be.e_contract.dssp.client.UnsupportedDocumentTypeException: null
        at be.e_contract.dssp.client.DigitalSignatureServiceClient.uploadDocument(DigitalSignatureServiceClient.java:329) ~[dssp-client-1.2.6.jar:na]
        at be.e_contract.dssp.client.DigitalSignatureServiceClient.uploadDocument(DigitalSignatureServiceClient.java:228) ~[dssp-client-1.2.6.jar:na]
        at net.awl.ecs.edoc.ap.domain.service.signing.DssSigningProvider.startSigningSession(DssSigningProvider.java:64) ~[mci-ap-core-5.1.3.jar:na]
        at net.awl.ecs.edoc.ap.domain.service.signing.SigningService.doStartSigningSession(SigningService.java:166) [mci-ap-core-5.1.3.jar:na]
```

It works fine for "application/pdf".
MIME types should be treated case insensitive! (http://stackoverflow.com/questions/12869278/are-mimetypes-lowercase-case-insensitive)

**Comment by fcorneli** [comment]
RFC 2045 refers to Content-Type as HTTP header, which is not the same at the MimeType attribute as defined within:
http://docs.oasis-open.org/dss/v1.0/oasis-dss-core-spec-v1.0-os.html
which is the specification that we implement here. OASIS DSS Core 1.0 also does not specify anything related to case sensitivity, nor does it refer to RFC 2045.

However, if this would be of any help, we can always change towards case insensitive handling of the received MimeType value within the next version of DSS. Just let us know.

**Comment by klr8** [comment]
> However, if this would be of any help, we can always change towards case insensitive handling of the received MimeType value within the next version of DSS. 

That would indeed be useful. We receive these MIME types from our clients and we could lowercase them before passing them on to DSS but that seems like a bit of a work-around.

**Comment by fcorneli** [comment]
Fix will be part of the DSS 1.7.13 release.
